spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Movie Details Page', type: :feature do
+  describe 'when a user visits a movies details page' do
```

Comment:
I am not sure if this would work, but could we move the `:vcr` to this line and remove it from each of the `it` blocks?